src/scss/2022/tools/_typography.scss

```diff
@@ -0,0 +1,12 @@
+@mixin style($size, $weight: 200) {
+  @if $size == 'heading-1'
+   or $size == 'heading-2'
+   or $size == 'heading-3'
+   or $size == 'heading-4'
+   or $size == 'section-description' {
+    font-family: var(--font-display);
+  }
```

Comment:
Everything LGTM! I can compile and run things locally on my M1 with @bengoshow changes [here](https://github.com/sparkbox/designsystemssurvey/pull/336)

That said when I was looking through the Figma design and what was computed in localhost. I saw that the line-heights were not matching up in the H1, H2, H4, and one or two of the Body typographies (off by a few ```px```). Also, the font-size was off by a few ```px``` too. 

Not sure if that is worth holding up the PR. Thoughts on this?
Side note : Im new to working within design systems

Reply:
@QuinnrElder that is a good catch, I like your attention to detail. One of the things you will hear me quote often is Dan Mall’s statement about design called “deciding in the browser.” Typography changes occur once the content starts to show up in the browser, so `line-height` especially, I keep a few things broad until further along in development.

Reply:
@zastrow ~ thank you for taking the time to go over that with me! You rock!